ckan/controllers/package.py

```diff
@@ -230,7 +230,7 @@ def pager_url(q=None, page=None):
             # types any search page. Potential alternatives are do show them
             # on the default search page (dataset) or on one other search page
             search_all_type = config.get(
-                'ckan.search.show_all_types', 'dataset')
+                'ckan.search.show_all_types', False)
```

Comment:
I've applied the same change here in the `ckan/controllers/package.py` controller as I did in the blueprint in `ckan/ckan/views/dataset.py`. The blueprint is the the one that actually gets called under test. I'm not sure about this one. Is it just to make back-porting to 2.8 easier? Is there some circumstance in 2.9 where the old controller still gets called?

Reply:
Core doesn't use these controllers at all, and in fact they will be removed soon, but we might backport this to 2.8 so it will make things easier to have this change here